upgrading/automated_upgrades.adoc

```diff
@@ -20,300 +20,155 @@ endif::[]
-* The default router, if one exists, is updated from 3.9 to 3.10.
-* The default registry, if one exists, is updated from 3.9 to 3.10.
-* The default image streams and InstantApp templates are updated.
+* Back up all etcd data for recovery purposes.
+* Update the API and controllers from 3.10 to 3.11.
+* Update internal data structures to 3.11.
+* Update the default router, if one exists, from 3.10 to 3.11.
+* Update the default registry, if one exists, from 3.10 to 3.11.
+* Update the default image streams and InstantApp templates.
 
-The 3.9 to 3.10 node upgrade performs a rolling update of nodes, which:
+The 3.10 to 3.11 node upgrade performs a rolling update of nodes, which:
 
 * Marks a subset of nodes unschedulable and drains them of pods.
-* Updates node components from 3.9 to 3.10.
-* Converts from local configuration to bootstrapped TLS and Config.
-* Converts SDN components from *systemd* services to DaemonSets.
+* Updates node components from 3.10 to 3.11.
 * Returns those nodes to service.
 
-[IMPORTANT]
-====
-* Ensure that you have met all
-xref:../install/prerequisites.adoc#install-config-install-prerequisites[prerequisites]
-before proceeding with an upgrade. Failure to do so can result in a failed
-upgrade.
+[[upgrade-prerequisites]]
+== Prerequisites
 
-* If you are using GlusterFS, see
-xref:special-considerations-for-glusterfs[Special Considerations When Using
-Containerized GlusterFS] before proceeding.
+Before you upgrade your cluster:
 
-* If you are using GCE Persistent Disk (gcePD), see xref:special-considerations-for-gcepd[Special Considerations When Using
-gcePD] before proceeding.
+* Upgrade the cluster to the
+link:https://docs.openshift.com/container-platform/3.10/release_notes/ocp_3_10_release_notes.html#ocp-3-10-34[latest asynchronous release of version 3.10].
+If your cluster is at a version earlier
+than 3.10, you must first upgrade incrementally. For example, upgrade from 3.7
+to 3.9 (the 3.8 version was link:https://docs.openshift.com/container-platform/3.9/release_notes/ocp_3_9_release_notes.html#ocp-39-about-this-release[skipped]),
+and then from 3.9 to 3.10.
+
+* Run the
+xref:../day_two_guide/environment_health_checks.adoc#day-two-environment-health-checks[Environment
+health checks] to verify the cluster's health. In this process, you confirm that
+the nodes are in the *Ready* state and running the expected starting version and
+that there are no diagnostic errors or warnings.
+
+* Ensure that your cluster meets the current
+xref:../install/prerequisites.adoc#install-config-install-prerequisites[prerequisites]
+If it does not, your upgrade might fail.
 
 * The day before the upgrade, validate {product-title} storage migration to ensure
-potential issues are resolved prior to the outage window:
+potential issues are resolved before the outage window:
 +
 ----
 $ oc adm migrate storage --include=* --loglevel=2 --confirm --config /etc/origin/master/admin.kubeconfig
 ----
 
```

Comment:
For v3.10 to v3.11 upgrade, there should be a matched node for Cluster Monitoring Operator which would be deployed by default. Or else, the upgrade will be broken due to no scheduled nodes found matching node selector for Cluster Monitoring Operator. So before upgrade, another prerequisite is that ensure you have infra node in the cluster(infra node is the default node for Cluster Monitoring Operator), if there is not infra node or user want to schedule it on other node, they need to specify the targeted node with openshift_cluster_monitoring_operator_node_selector, or if user did not want it deployed, then they need to disable it by specifying openshift_cluster_monitoring_operator_install=false.

Reply:
or you can add this description in following section "Review and update your inventory file."